src/pages/Admin/organizations/components/AdminOrganizationFormSheet.tsx

```diff
@@ -210,7 +210,11 @@ export default function AdminOrganizationFormSheet({
               )}
             />
 
-            <Button type="submit" className="w-full" disabled={isPending}>
+            <Button
+              type="submit"
+              className="w-full"
+              disabled={isPending || (isEditMode && !form.formState.isDirty)}
```

Comment:
why only for edit? 🤔 

we can disable it even when it's create and form is not dirty right? 🤔